Added decimal field to keystone-next with precision, scale (#4938)
keystonejs/keystone: .changeset/good-melons-whisper.md

---
'@keystone-next/fields': minor
'@keystonejs/fields': minor
---

> Added decimal field type for keystone-next with precision, scale

Reviewer: ```suggestion
Added a `decimal` field type to keystone-next.
```
